sympy/polys/tests/test_polytools.py

```diff
@@ -1993,6 +1995,12 @@ def test_gcd():
     assert gcd(f, g, modulus=11, symmetric=False) == h
     assert lcm(f, g, modulus=11, symmetric=False) == l
 
+    a, b = sqrt(2), -sqrt(2)
```

Comment:
It might be a good idea to confirm that the reverse order passes, too. (And it does with this PR.)